### ISSUE TITLE
【EESS】New submissions for Wed, 14 Jun 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#303]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Speech-to-Text Adapter and Speech-to-Entity Retriever Augmented LLMs for  Speech Understanding
 - **Authors:** Mingqiu Wang, Izhak Shafran, Hagen Soltau, Wei Han, Yuan Cao, Dian Yu, Laurent El Shafey
 - **Subjects:** Audio and Speech Processing (eess.AS); Artificial Intelligence (cs.AI); Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2306.07944
 - **Pdf link:** https://arxiv.org/pdf/2306.07944
 - **Abstract**
 Large Language Models (LLMs) have been applied in the speech domain, often incurring a performance drop due to misaligned between speech and language representations. To bridge this gap, we propose a joint speech and language model (SLM) using a Speech2Text adapter, which maps speech into text token embedding space without speech information loss. Additionally, using a CTC-based blank-filtering, we can reduce the speech sequence length to that of text. In speech MultiWoz dataset (DSTC11 challenge), SLM largely improves the dialog state tracking (DST) performance (24.7% to 28.4% accuracy). Further to address errors on rare entities, we augment SLM with a Speech2Entity retriever, which uses speech to retrieve relevant entities, and then adds them to the original SLM input as a prefix. With this retrieval-augmented SLM (ReSLM), the DST performance jumps to 34.6% accuracy. Moreover, augmenting the ASR task with the dialog understanding task improves the ASR performance from 9.4% to 8.5% WER.
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### PauseSpeech: Natural Speech Synthesis via Pre-trained Language Model and  Pause-based Prosody Modeling
 - **Authors:** Ji-Sang Hwang, Sang-Hoon Lee, Seong-Whan Lee
 - **Subjects:** Audio and Speech Processing (eess.AS); Artificial Intelligence (cs.AI); Sound (cs.SD); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2306.07489
 - **Pdf link:** https://arxiv.org/pdf/2306.07489
 - **Abstract**
 Although text-to-speech (TTS) systems have significantly improved, most TTS systems still have limitations in synthesizing speech with appropriate phrasing. For natural speech synthesis, it is important to synthesize the speech with a phrasing structure that groups words into phrases based on semantic information. In this paper, we propose PuaseSpeech, a speech synthesis system with a pre-trained language model and pause-based prosody modeling. First, we introduce a phrasing structure encoder that utilizes a context representation from the pre-trained language model. In the phrasing structure encoder, we extract a speaker-dependent syntactic representation from the context representation and then predict a pause sequence that separates the input text into phrases. Furthermore, we introduce a pause-based word encoder to model word-level prosody based on pause sequence. Experimental results show PauseSpeech outperforms previous models in terms of naturalness. Furthermore, in terms of objective evaluations, we can observe that our proposed methods help the model decrease the distance between ground-truth and synthesized speech. Audio samples are available at https://jisang93.github.io/pausespeech-demo/.
### Approximate Maximum-Likelihood RIS-Aided Positioning
 - **Authors:** Wei Zhang, Zhenni Wang, Wee Peng Tay
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2306.07504
 - **Pdf link:** https://arxiv.org/pdf/2306.07504
 - **Abstract**
 A reconfigurable intelligent surface (RIS) allows a reflection transmission path between a base station (BS) and user equipment (UE). In wireless localization, this reflection path aids in positioning accuracy, especially when the line-of-sight (LOS) path is subject to severe blockage and fading. In this paper, we develop a RIS-aided positioning framework to locate a UE in environments where the LOS path may or may not be available. We first estimate the RIS-aided channel parameters from the received signals at the UE. To infer the UE position and clock bias from the estimated channel parameters, we propose a fusion method consisting of weighted least squares over the estimates of the LOS and reflection paths. We show that this approximates the maximum likelihood estimator under the large-sample regime and when the estimates from different paths are independent. We then optimize the RIS phase shifts to improve the positioning accuracy and extend the proposed approach to the case with multiple BSs and UEs. We derive Cramer-Rao bound (CRB) and demonstrate numerically that our proposed positioning method approaches the CRB.
### Statistical Beamformer Exploiting Non-stationarity and Sparsity with  Spatially Constrained ICA for Robust Speech Recognition
 - **Authors:** Ui-Hyeop Shin, Hyung-Min Park
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2306.07562
 - **Pdf link:** https://arxiv.org/pdf/2306.07562
 - **Abstract**
 In this paper, we present a statistical beamforming algorithm as a pre-processing step for robust automatic speech recognition (ASR). By modeling the target speech as a non-stationary Laplacian distribution, a mask-based statistical beamforming algorithm is proposed to exploit both its output and masked input variance for robust estimation of the beamformer. In addition, we also present a method for steering vector estimation (SVE) based on a noise power ratio obtained from the target and noise outputs in independent component analysis (ICA). To update the beamformer in the same ICA framework, we derive ICA with distortionless and null constraints on target speech, which yields beamformed speech at the target output and noises at the other outputs, respectively. The demixing weights for the target output result in a statistical beamformer with the weighted spatial covariance matrix (wSCM) using a weighting function characterized by a source model. To enhance the SVE, the strict null constraints imposed by the Lagrange multiplier methods are relaxed by generalized penalties with weight parameters, while the strict distortionless constraints are maintained. Furthermore, we derive an online algorithm based on an optimization technique of recursive least squares (RLS) for practical applications. Experimental results on various environments using CHiME-4 and LibriCSS datasets demonstrate the effectiveness of the presented algorithm compared to conventional beamforming and blind source extraction (BSE) based on ICA on both batch and online processing.
### Rethinking Polyp Segmentation from an Out-of-Distribution Perspective
 - **Authors:** Ge-Peng Ji, Jing Zhang, Dylan Campbell, Huan Xiong, Nick Barnes
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2306.07792
 - **Pdf link:** https://arxiv.org/pdf/2306.07792
 - **Abstract**
 Unlike existing fully-supervised approaches, we rethink colorectal polyp segmentation from an out-of-distribution perspective with a simple but effective self-supervised learning approach. We leverage the ability of masked autoencoders -- self-supervised vision transformers trained on a reconstruction task -- to learn in-distribution representations; here, the distribution of healthy colon images. We then perform out-of-distribution reconstruction and inference, with feature space standardisation to align the latent distribution of the diverse abnormal samples with the statistics of the healthy samples. We generate per-pixel anomaly scores for each image by calculating the difference between the input and reconstructed images and use this signal for out-of-distribution (ie, polyp) segmentation. Experimental results on six benchmarks show that our model has excellent segmentation performance and generalises across datasets. Our code is publicly available at https://github.com/GewelsJI/Polyp-OOD.
### Deep Learning-Enabled Zero-Touch Device Identification: Mitigating the  Impact of Channel Variability Through MIMO Diversity
 - **Authors:** Bechir Hamdaoui, Nora Basha, Kathiravetpillai Sivanesan
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2306.07878
 - **Pdf link:** https://arxiv.org/pdf/2306.07878
 - **Abstract**
 Deep learning-enabled device fingerprinting has proven efficient in enabling automated identification and authentication of transmitting devices. It does so by leveraging the transmitters' unique features that are inherent to hardware impairments caused during manufacturing to extract device-specific signatures that can be exploited to uniquely distinguish and separate between (identical) devices. Though shown to achieve promising performances, hardware fingerprinting approaches are known to suffer greatly when the training data and the testing data are generated under different channels conditions that often change when time and/or location changes. To the best of our knowledge, this work is the first to use MIMO diversity to mitigate the impact of channel variability and provide a channel-resilient device identification over flat fading channels. Specifically, we show that MIMO can increase the device classification accuracy by up to about $50\%$ when model training and testing are done over the same channel and by up to about $70\%$ when training and testing are done over different fading channels.
### A Theory of Unsupervised Speech Recognition
 - **Authors:** Liming Wang, Mark Hasegawa-Johnson, Chang D. Yoo
 - **Subjects:** Audio and Speech Processing (eess.AS); Computation and Language (cs.CL); Machine Learning (cs.LG); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2306.07926
 - **Pdf link:** https://arxiv.org/pdf/2306.07926
 - **Abstract**
 Unsupervised speech recognition (ASR-U) is the problem of learning automatic speech recognition (ASR) systems from unpaired speech-only and text-only corpora. While various algorithms exist to solve this problem, a theoretical framework is missing from studying their properties and addressing such issues as sensitivity to hyperparameters and training instability. In this paper, we proposed a general theoretical framework to study the properties of ASR-U systems based on random matrix theory and the theory of neural tangent kernels. Such a framework allows us to prove various learnability conditions and sample complexity bounds of ASR-U. Extensive ASR-U experiments on synthetic languages with three classes of transition graphs provide strong empirical evidence for our theory (code available at cactuswiththoughts/UnsupASRTheory.git).
